docs/upgrade.html

```diff
@@ -97,6 +97,9 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
         <code>DefaultReplicationPolicy</code>, cannot prevent replication cycles based on topic names, so take care to avoid cycles when constructing your
         replication topology.
     </li>
+    <li> The original MirrorMaker (MM1) and related classes have been deprecated. Please use the Connect-based
+        MirrorMaker (MM2), as described in <a href="/{{version}}/documentation/#georeplication">.
```

Comment:
`<a href="/{{version}}/documentation/#georeplication">` should be `the <a href="/{{version}}/documentation/#georeplication">Geo-Replication section</a>`

Reply:
lol whoops

Reply:
fixed, thx